### PR TITLE
Add border to neovim floating window.

### PR DESCRIPTION
**Pull request by kristijanhusak**
This adds support for borders for neovim floating window. Also fixes part of https://github.com/pechorin/any-jump.vim/issues/27.

### CODE REVIEW

**Comment by pechorin** [comment]
Please provide screenshot for such big improvment, also where should be way to configure borders

**Comment by kristijanhusak** [comment]
Here's a screenshot:
screenshot

And I made it configurable:
```vim
" top left corner, top right corner, bottom left corner, bottom right corner, top/bottom, left/right
let g:any_jump_neovim_border = ['╭', '╮','╰', '╯', '─', '│']
```

**Comment by georgijd** [comment]
Hi @pechorin,

Awesome project! Have you considered merging this feature in the near future? 🙏🏼

**Comment by numbcoder** [comment]
ping @pechorin

**Comment by pechorin** [comment]
I think this possible if this will be option (and not enabled by default). You can rework this, or wait for me :)

File: doc/any-jump.txt [hunk 1]
```diff
@@ -137,3 +137,7 @@ let g:any_jump_references_only_for_current_filetype = 0
 
 " Disable search engine ignore vcs untracked files (default: false, search engine will ignore vcs untracked files)
 let g:any_jump_disable_vcs_ignore = 0
+
+" Borders for neovim floating window, list of strings in this order
+" top left corner, top right corner, bottom left corner, bottom right corner, top/bottom, left/right
+let g:any_jump_neovim_border = ['╭', '╮','╰', '╯', '─', '│']
```
Comment (pechorin) [inline doc/any-jump.txt:143]: Please add option, for enabling or disabling this (this should not be enabled by default)

File: plugin/any-jump.vim [hunk 2]
```diff
@@ -213,8 +215,24 @@ fu! s:CreateNvimUi(internal_buffer) abort
         \ 'style': 'minimal',
         \ }
 
+  let [topleft_b, topright_b, botleft_b, botright_b, horizontal_b, vertical_b] = g:any_jump_neovim_border
```
Comment (pechorin) [inline plugin/any-jump.vim:218]: Please move all this logic code part to separate function :)

**Comment by kristijanhusak** [comment]
This is not applicable any more because neovim has built in borders now. It should be much simpler to add it. I'll let someone else take care of that.

**Pull request closed without merging**